src/css/base.scss

```diff
@@ -74,4 +94,133 @@ body {
   background-color: $light-blue;
   padding-left: 10px;
   font-size: 20px;
+}
+
+.main__entering-names-header {
+  color: #fff;
+  font-size: 20px;
+} 
+
+// part two of the dome > Game part
+
+.main__game-background {
+  width: 100%;
+  // height: 100vh;
+  border: 1px solid transparent
+}
+
+.main__game-board-container {
+  width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 80px;
+  border-radius: 4px;
+}
+
+.main__game-board {
+  border: 1px solid #000;
+  width: 100%;
+  height: 450px;
+  
+}
+table {
+  border-collapse: collapse;
+  table-layout: fixed;
+}
+tr {
+  width: 100%;
+  height: 20%;
+  background-color: $dark-blue;
+}
+
+th, td {
+  border: 2px solid black;
+}
+th {
+  color: #fff;
```

Comment:
What's the reason for only storing some colors in variables?

Reply:
@thatPamIAm we only added the three main colors of our theme as variables, because we keep on using these three colors on our page more than other colors.

Reply:
Cool - that makes sense @hndfaw !